lib/include/openamp/virtio.h

```diff
@@ -111,6 +111,24 @@ struct virtio_buffer_info {
 	unsigned long size;
 };
 
+/**
+ * struct remoteproc_vring - remoteproc vring structure
+ * @vq virtio queue
+ * @va logical address
+ * @notifyid vring notify id
+ * @num_descs number of descriptors
+ * @align vring alignment
+ * @io metal I/O region of the vring memory, can be NULL
+ */
+struct virtio_vring {
+	struct virtqueue *vq;
+	void *va;
```

Comment:
why void *, why not struct vring *va;

Reply:
i rename this virtio_vring to vritio_vring_info, the va here is about the vrings area.
However, this struct is not aware of any specific data struct in the vrings area, this information will be passed to vq_ring_init() which will initialize the vring area.